app/controllers/authenticate_controller.rb

```diff
@@ -223,6 +225,20 @@ def handle_authentication_error(err)
     end
   end
 
+  def audit_authn_error(err)
+    authn_info = authenticator_input
```

Comment:
this seems redundant. if you think that `authn_info` is a better name than `authenticator_input` then let's replace the method itself. Or else just leave it as `authenticator_input`